### PR TITLE
Get all posts by sender ID

### PR DESCRIPTION
**Pull request by Michalikhol**

### CODE REVIEW

File: .gitignore
Comment (shirshponder) [inline .gitignore:1]: whats this

File: src/controllers/postsController.js [hunk 1]
```diff
@@ -1,19 +1,22 @@
...
+    const senderId = req.query.sender;
+    var searchQuery;
+
+    if (senderId) {
+      searchQuery = { sender: senderId };
     }
+
+    const posts = await postModel.find(searchQuery);
+
+    res.status(status.OK).send(posts);
```
Comment (shirshponder) [inline src/controllers/postsController.js]: maybe better?

` const posts = await postModel.find(senderId && { sender: senderId });
    res.status(status.OK).send(posts);`

Comment (Michalikhol) [reply src/controllers/postsController.js]: yes better

File: .gitignore
Comment (Michalikhol) [reply .gitignore:1]: added to git ignore file so we will not upload settings and more stuff

File: src/controllers/postsController.js [hunk 2]
```diff
@@ -1,19 +1,22 @@
...
   } catch (error) {
-    res.status(status.BAD_REQUEST).send(error.message);
+    res
+      .status(status.BAD_REQUEST)
+      .send({ status: 'fail', message: error.message });
```
Comment (shirshponder) [inline src/controllers/postsController.js]: think you can remove status:fail

**Pull request merged**